.appveyor.yml

```diff
@@ -38,9 +38,9 @@ environment:
   matrix:
 
     - PYTHON: "C:\\Python37_64"
-      PYTHON_VERSION: "3.7"
+      PYTHON_VERSION: "3.9"
```

Comment:
What is the reason to require Python 3.9?

Reply:
With 3.7 the tests did not work, so I tried updating. No help, it seems there are some problems related to using conda.

Reply:
This is a very strict requirement. The latest LTS Ubuntu 20.4 has Python 3.8.

Reply:
We can change it to 3.8, as it does not work anyway :)